docs/guide/content-script-ui.md

```diff
@@ -21,7 +21,7 @@ Integrated content script UIs are injected alongside the content of a page. This
 ```ts [Vanilla]
 // entrypoints/example-ui.content.ts
 export default defineContentScript({
-  matches: ['*:///*/*'],
+  matches: ['<all_urls>'],
```

Comment:
Ahh 🤦 one to many slashes, should have been `*://*/*`. All urls is fine for the docs too.